packages/common/src/store/pages/collection/lineup/reducer.ts

```diff
@@ -9,15 +9,25 @@ export const initialState = {
   prefix: PREFIX
 }
 
+type ResetSucceededAction = {
+  type: typeof RESET_SUCCEEDED
+}
+
 const actionsMap = {
-  [RESET_SUCCEEDED](state, action) {
+  [RESET_SUCCEEDED](
+    _state: LineupState<Collection>,
+    _action: ResetSucceededAction
+  ) {
     const newState = initialState
     return newState
   }
 }
 
-const tracks = (state = initialState, action) => {
-  const baseActionType = stripPrefix(PREFIX, action.type)
+const tracks = (state = initialState, action: ResetSucceededAction) => {
+  const baseActionType = stripPrefix(
+    PREFIX,
+    action.type
+  ) as typeof RESET_SUCCEEDED
```

Comment:
not really sure how this `baseType` works

Reply:
this is just to pass along just the `RESET_SUCCEEDED` part and not the full `PAGES/COLLECTION/RESET_SUCCEEDED` action